setup.cfg

```diff
@@ -1,15 +1,15 @@
 [metadata]
 name = hip
-description = HTTP library with thread-safe connection pooling, file post, and more.
+description = Hip: A new Python HTTP client for Everyone
 long_description = file: README.rst, CHANGES.rst
-keywords = urllib, httplib, threadsafe, filepost, http, https, ssl, pooling
-license = MIT
-author = Andrey Petrov
-author_email = andrey.petrov@shazow.net
-url = https://hip.readthedocs.io/
+keywords = http client
+license = MIT -or- Apache-2.0
+author = Nathaniel J. Smith
+author_email = njs@pobox.com
```

Comment:
I feel a little weird claiming authorship here... I know normally the author field is like "yeah uh this is who first setup the package and they might or might not have anything to do with it anymore", but here I'm already probably not the biggest contributor at the time when we're setting the field :-)

Maybe we should leave this out, or put something like "The Hip project maintainers" or "The Trio Collective"?

Reply:
+1 for "The Trio Collective". Do we need to keep the email?

Reply:
+1 for "The Trio Collective".

Reply:
I also like Trio Collective so I'll use that. For now I can keep your email address @njsmith as it is a "recommended" field.